docs/HowToAddnewBoard.md

```diff
@@ -2,17 +2,17 @@
 
 To support a new board, you should make changes on three folders, *arch*, *board* and *configs*.  
 
-> 1. **arch** folder includes *CPU* and *chip* architectures.  
-> 2. **board** folder includes board support package(BSP).  
-> 3. **configs** folder includes build options and flash operations.
+- **arch** folder includes *CPU* and *chip* architectures.
+- **board** folder includes board support package(BSP).
+- **configs** folder includes build options and flash operations.
```

Comment:
It should be done by the order as previous. I think `1. 2. 3.` is necessary.

Reply:
As per the suggestion, they are not sequential contents (you could change them in any order), so numbering isn't necessary. Please let me know if you still prefer sequential ordering.

Reply:
No. it is sequential content. Suggestion is wrong.

Reply:
Alright, I will re-introduce sequence. However, BlockQuote has to be replaced by unordered lists, as per suggestion. Thank  you.